### ISSUE TITLE
Recent update?

### ISSUE THREAD

**Issue by Sbozzolo** [adsabs/adsabs-dev-api#74]
Recently, `bibslurp` stopped working (https://github.com/mkmcc/bibslurp/issues/12#issuecomment-833763778). I suspect that something changed in the APIs. I think that the following used to work:
```
curl -d "bibcode=2010PhRvD..82j4014W" -H 'Authorization: Bearer MY_TOKEN' 'https://api.adsabs.harvard.edu/v1/export/bibtex'
```
but now it doesn't anymore and I need to send the data as `'-d {"bibcode":"2010PhRvD..82j4014W"}`.

I cannot find any announcement about any change. Has there been any change on the server side?

**Comment by csgrant00** [comment]
Hi.

Curl works with export service. bibcode is actually a list, even if you are
requesting one bibcode only. So this works

curl -d '{"bibcode":["2010PhRvD..82j4014W"]}' -H 'Authorization: Bearer
token' 'https://api.adsabs.harvard.edu/v1/export/bibtex'


{"msg": "Retrieved 1 abstracts, starting with number 1.", "export":
"@ARTICLE{2010PhRvD..82j4014W,\n       author = {{Witek}, Helvi and
{Zilh{\\~a}o}, Miguel and {Gualtieri}, Leonardo and {Cardoso}, Vitor and
{Herdeiro}, Carlos and {Nerozzi}, Andrea and {Sperhake}, Ulrich},\n
    title
= \"{Numerical relativity for D dimensional space-times: Head-on collisions
of black holes and gravitational wave extraction}\",\n      journal =
{\\prd},\n     keywords = {04.25.dg, 04.50.Gh, Numerical studies of black
holes and black-hole binaries, Higher-dimensional black holes black strings
and related objects, General Relativity and Quantum Cosmology, Astrophysics
- High Energy Astrophysical Phenomena, High Energy Physics - Phenomenology,
High Energy Physics - Theory},\n         year = 2010,\n        month =
nov,\n       volume = {82},\n       number = {10},\n          eid =
{104014},\n        pages = {104014},\n          doi =
{10.1103/PhysRevD.82.104014},\narchivePrefix = {arXiv},\n       eprint =
{1006.3081},\n primaryClass = {gr-qc},\n       adsurl = {
https://ui.adsabs.harvard.edu/abs/2010PhRvD..82j4014W},\n      adsnote =
{Provided by the SAO/NASA Astrophysics Data System}\n}\n\n"}(python)

golnaz

**Comment by Sbozzolo** [comment]
I would like to understand why `bibslurp` stopped working and I can prevent it from breaking again in the future. What we are doing in `bibslurp` is what I reported in the first message. I suspect that there has been a recent server-side change and now what we are doing in `bibslurp` is not allowed anymore. Is this the case?

**Comment by Sbozzolo** [comment]
Thanks, with your comments I updated `bibslurp` so that it works again. I realized that we were using the APIs incorrectly and `bibslurp` should not have worked in the first place (but it has been working fine for two years!). I cannot think of any other explanation for why it stopped working except that something changed server-side. (I cannot say how recently the package stopped working, it could have been in the past 2/3 months maybe?).

I guess this issue is solved. Thanks!
